internal/kafka/internal/config/kafka_supported_instance_types.go

```diff
@@ -35,6 +35,7 @@ type KafkaBillingModel struct {
 	AMSResource      string   `yaml:"ams_resource" validate:"required,ams_resource_validator"`
 	AMSProduct       string   `yaml:"ams_product" validate:"required,ams_product_validator"`
 	AMSBillingModels []string `yaml:"ams_billing_models" validate:"min=1,unique,ams_billing_models_validator"`
+	GracePeriodDays  int      `yaml:"grace_period_days"`
```

Comment:
Here we can add a minimum validation of 0

Reply:
Unless we want to use `-1` to signal another meaning?

Reply:
don't think we need a `-1` if we have 0, i'll add `validate:"min=0"` to this, thanks!